### ISSUE TITLE
Can't install wallabag via composer install if my mysql database doesn't exist

### ISSUE THREAD

**Issue by nicosomb** [wallabag/wallabag#1769]
### Issue details

When I launch `SYMFONY_ENV=prod composer install --no-dev -o`, if my mysql database doesn't exist before this command, I can't install wallabag: 

```
An exception occured in driver: SQLSTATE[42000] [1049] Unknown database 'wallabag' 
``` 

Found with @chris25640 

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.0.0-beta.1
* How did you install wallabag? Via `composer install`
* Last wallabag version that did not exhibit the issue (if applicable): N/A
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MySQL

### Steps to reproduce/test case

* git clone https://github.com/wallabag/wallabag.git
* cd wallabag
* git checkout 2.0.0-beta.1
* SYMFONY_ENV=prod composer install --no-dev -o
* Set `pdo_mysql` for `database_driver` parameter
* Set a name of an inexistent database for `database_name`

**Comment by nicosomb** [comment]
See http://stackoverflow.com/questions/27503328/composer-install-fails-when-unable-to-see-mysql-database and https://github.com/doctrine/DoctrineBundle/issues/351#issuecomment-75456547

**Comment by captainwasabi** [comment]
I added the line in apt/config/config.yml and I still get the same error when trying to set up beta 2

**Comment by nicosomb** [comment]
Fixed by #1903.
